api4/bot.go

```diff
@@ -30,8 +31,9 @@ func (api *API) InitBot() {
 }
 
 func createBot(c *Context, w http.ResponseWriter, r *http.Request) {
-	botPatch := model.BotPatchFromJson(r.Body)
-	if botPatch == nil {
+	var botPatch *model.BotPatch
+	err := json.NewDecoder(r.Body).Decode(&botPatch)
+	if err != nil {
```

Comment:
You really don't like the short form do you? :stuck_out_tongue:

Reply:
It's not intentional :laughing: That's just how I think of writing the code.